src/utils/helpers/number.ts

```diff
@@ -162,7 +163,17 @@ export function prettifyNumber(
   } else {
     intlOptions.maximumSignificantDigits = 5;
   }
-  const formatter = new Intl.NumberFormat(locale, intlOptions);
+  let formatter: Intl.NumberFormat;
+  try {
+    formatter = new Intl.NumberFormat(locale, intlOptions);
+  } catch (e) {
+    if (intlOptions.minimumFractionDigits) {
+      intlOptions.maximumFractionDigits = intlOptions.minimumFractionDigits + 1;
+    }
+    setContext('prettifyNumber', intlOptions);
```

Comment:
this will send us the intlOptions after they were modified. 
Please move the sentry calls to before the `if`